### PR TITLE
Change label for bug reports

### PR DESCRIPTION
**Pull request by jasonnutter**
Currently, bug reports are given the bug label by default. Proposing bug reports are marked as questions until they are confirmed as bugs.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 name: Bug report
 about: Broken or unintended behavior with one of our libraries.
 title: ''
-labels: bug
+labels: question
```
Comment (jasonnutter) [inline .github/ISSUE_TEMPLATE/bug_report.md:5]: We could also add another label such as "unconfirmed bug", which gets replaced with "bug" once confirmed.

Comment (tnorling) [reply .github/ISSUE_TEMPLATE/bug_report.md:5]: I like this option, start with `Bug: Unconfirmed` and then manually change to `Question` and `Bug: Confirmed` or just `Bug`

Big fan of changing this regardless of what we decide to change it to!

Comment (sameerag) [reply .github/ISSUE_TEMPLATE/bug_report.md:5]: I am leaning towards "unconfirmed" too instead of "question". +1 for changing this.

Comment (jo-arroyo) [reply .github/ISSUE_TEMPLATE/bug_report.md:5]: +1 on unconfirmed!

Comment (jasonnutter) [reply .github/ISSUE_TEMPLATE/bug_report.md:5]:
```suggestion
labels: bug-unconfirmed
```

**Review by jo-arroyo — APPROVED** [review]

**Review by tnorling — APPROVED** [review]

**Pull request merged**